I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/game_content/apprentices_den.rpy", line 269, in script
    $ herbs_clicked += 1
  File "game/scripts/game_content/apprentices_den.rpy", line 269, in <module>
    $ herbs_clicked += 1
  File "game/scripts/defines/achievement_backend.rpy", line 206, in add_progress
    self.progress(min(self.stat_progress+amount, self.stat_max))
TypeError: '<' not supported between instances of 'NoneType' and 'int'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/game_content/apprentices_den.rpy", line 269, in script
    $ herbs_clicked += 1
  File "C:\Users\annie\Downloads\renpy-8.3.4-sdk\renpy\ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\annie\Downloads\renpy-8.3.4-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/scripts/game_content/apprentices_den.rpy", line 269, in <module>
    $ herbs_clicked += 1
  File "game/scripts/defines/achievement_backend.rpy", line 206, in add_progress
    self.progress(min(self.stat_progress+amount, self.stat_max))
TypeError: '<' not supported between instances of 'NoneType' and 'int'

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.4.24120703
 1.0
Fri Apr 11 18:09:41 2025
